server.js

```diff
@@ -659,6 +659,37 @@ router.get("/api/repos", function (req, res) {
         );
     });
 });
+
+// list contributors to a repo
+router.get("/api/repos/:owner/:shortName/contributors", function (req, res) {
```

Comment:
I think we should add tests for that new API endpoint :)

Reply:
I added 2 tests for that endpoint